Dockerfile

```
@@ -0,0 +1,7 @@
+FROM python:3.10
```

```suggestion
FROM python:3.11.4
```

or 

```suggestion
FROM python:3.10.12
```

If you want to stick with 3.10.

Its a bit of paranoia, but want to make sure people have some recent security fixes.